subgraph-mainnet-production.yaml

```diff
@@ -126,4 +126,83 @@ dataSources:
       callHandlers:
         - function: file(bytes32,uint256)
           handler: handleSeniorTrancheFile
-      file: ./src/mapping.ts
\ No newline at end of file
+      file: ./src/mapping.ts
+  - kind: ethereum/contract
+    name: Shelf2
+    network: mainnet
+    source:
+      address: "0x3d0e477F328E48dAA315aA503a6eDf5B67f2d387"
+      abi: Shelf
+      startBlock: 10103234
```

Comment:
Are you sure this Startblock (and below) is correct? They are different from the ones in the poolMetas

Reply:
yes in the pool metas I took the startblock of the root contract :) It is always a bit ahead